src/components/Footer.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="footer-container">
+    <footer>
+      <h4>Arrons kit</h4>
+      <p>Making research accessible</p>
+
+      <p>© 2021</p>
+    </footer>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.footer-container {
+  margin-top: auto;
+
+  footer {
+    margin-top: 5%;
```

Comment:
avoid using %. does weird things in flexbox.
consider removing
```margin-top: 5%```
and using 
```height: 4rem```

then go to  Home.vue and replace ```.container``` in style with
```
.container {
  display: flex;
  flex-grow: 1;
  flex-shrink: 0;
  flex-direction: column;
  justify-items: center;
  align-items: center;
  padding-top: 2rem;
}
```

the two lines: 
```   
flex-grow: 1;
flex-shrink: 0;
``` 
will make Home.vue stretch to fill remaining space  forcing footer to stick to bottom